channeldb/migration_01_to_11/migrations.go

```diff
@@ -724,7 +724,8 @@ func MigrateGossipMessageStoreKeys(tx kvdb.RwTx) error {
 
 		// Serialize the message with its wire encoding.
 		var b bytes.Buffer
-		if _, err := lnwire.WriteMessage(&b, msg, 0); err != nil {
+		_, err := lnwire.WriteMessage(&b, msg, lnwire.ProtocolVersionTLV)
```

Comment:
why TLV here?

Reply:
Hmm may have been a mistake during the rebase. Looking at it with fresh eyes though, it needs to ensure we write any extra bytes that may have been stored with the announcement.